### PR TITLE
Update builders to temporarily remove ubuntu24

### PR DESCRIPTION
**Pull request by johnmccrae**
## Description
We're not ready for Ubuntu 24.04 yet. Commenting this out for now.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] If `Gemfile.lock` has changed, I have used `--conservative` to do it and included the full output in the Description above.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Review by rahulgoel1 — APPROVED** [review]

**Comment by Stromweld** [comment]
Not sure if buildkite was failing, but the other kitchen tests were working for ubuntu 24.04. Looks like omnitruck is pulling it from the ubuntu 22.04 repo. Here is an example of that https://github.com/chef/chef/actions/runs/11220728684/job/31189536682.

File: kitchen-tests/kitchen.dokken.yml [hunk 1]
```diff
@@ -180,13 +180,13 @@ platforms:
...
-      intermediate_instructions:
-        - RUN /usr/bin/apt-get update -y
-        - RUN /usr/bin/apt-get install ifupdown -y # we need this for /etc/network/interfaces & ifconfig resource testing
+  # - name: ubuntu-24.04
+  #   driver:
+  #     image: dokken/ubuntu-24.04
+  #     pid_one_command: /bin/systemd
+  #     intermediate_instructions:
+  #       - RUN /usr/bin/apt-get update -y
+  #       - RUN /usr/bin/apt-get install ifupdown -y # we need this for /etc/network/interfaces & ifconfig resource testing
```
Comment (Stromweld) [inline kitchen-tests/kitchen.dokken.yml:189]: This is just a configuration item and doesn't need to be commented out. Only the one in the .github workflows file controls what is actually ran .github/workflows/kitchen.yml

File: kitchen-tests/kitchen.linux.ci.yml [hunk 2]
```diff
@@ -65,7 +65,7 @@ platforms:
   - name: ubuntu-18.04
   - name: ubuntu-20.04
   - name: ubuntu-22.04
-  - name: ubuntu-24.04
+  # - name: ubuntu-24.04
```
Comment (Stromweld) [inline kitchen-tests/kitchen.linux.ci.yml:68]: This is just a configuration item and doesn't need to be commented out. Only the one in the .github workflows file controls what is actually ran .github/workflows/kitchen.yml

File: kitchen-tests/kitchen.yml [hunk 3]
```diff
@@ -40,7 +40,7 @@ platforms:
   - name: ubuntu-18.04
   - name: ubuntu-20.04
   - name: ubuntu-22.04
-  - name: ubuntu-24.04
+  # - name: ubuntu-24.04
```
Comment (Stromweld) [inline kitchen-tests/kitchen.yml:43]: This is just a configuration item and doesn't need to be commented out. Only the one in the .github workflows file controls what is actually ran .github/workflows/kitchen.yml

**Review by Stromweld — CHANGES_REQUESTED** [review]

**Review by Stromweld — APPROVED** [review]
LGTM

**Pull request merged**